packages/fxa-auth-server/lib/senders/emails/partials/appBadges.ts

```diff
@@ -0,0 +1,21 @@
+export const appBadges = `<mj-section>
```

Comment:
I think the string literals are OK for the initial PR here, but we'll definitely want to create a follow up in this milestone to see what we can do here. This is not very dev-friendly 😝

Reply:
Yeah sure. I couldn't find another way of using `mjml` inside typescript, but sure we can do some digging around this down the line